manifest-list.md

```diff
@@ -64,6 +66,13 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
         This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature (for example `sse4` or `aes`).
 
+- **`elements`** *array of objects*
+
+  This OPTIONAL property contains a list of non-manifest objects.
```

Comment:
> I would think it much cleaner to just have a single field like descriptors where they all live. Then this desire to distinguish goes away.

I'm not sure we can make this work. Migrating these top-level types are a huge pain and I'm not sure we gain anything except naming. I'd prefer to call this `extensions`, as they seem like extensions.